frontend/src/__tests__/cypress/cypress/pages/loginDialog.ts

```diff
@@ -0,0 +1,9 @@
+import Chainable = Cypress.Chainable;
+
+export class LoginDialog {
+  findText(): Chainable<JQuery<HTMLElement>> {
+    return cy.findByTestId('timeout-text');
+  }
+}
```

Comment:
For modals we normally create a contextual page object which extends the `Modal` class:
```suggestion
class LoginDialog extends Modal {
  constructor() {
    super('Session Expired');
  }
}
```

With a new instance of this class you can then do `loginDialog().shouldBeOpen()` instead of the modal text assertion.

Reply:
Applied this change with `loginDialog.shouldBeOpen()` - the test isn't passing though because the `find()` call in `shouldBeOpen` doesn't identify this modal correctly. Investigating...